src/components/primitive/box/box.component.tsx

```diff
@@ -146,6 +149,19 @@ export const Box: React.FunctionComponent<BoxComponentProps> = (
     return <></>;
   }
 
+  if (loading) {
+    return (
+      <Base style={style} {...rest}>
+        <Oval
+          arialLabel="loading-indicator"
+          height={40}
+          width={40}
+          color={colors.primary}
+        />
```

Comment:
@cksadhukhan  you can make use of keyframes, instead of using a package here.

```
/**
 * KeyFrame Animation.
 */
const spin = keyframes`
    0% {
      transform: rotate(0deg);
    }
    100% {
       transform: rotate(360deg);
       }
`;

/**
 * Styles for Loading Spinner.
 */
export const Spinner = styled.div`
  border: 0.4rem solid ${({ theme: { colors } }) => colors.spinnerBorder};
  border-top: 0.4rem solid ${({ theme: { colors } }) => colors.primary};
  border-radius: 50%;
  width: 4rem;
  height: 4rem;
  animation: ${spin} 0.6s linear infinite;
`;

```
This will work.

Reply:
This is inside 'notice-loader.component.styles file.

Reply:
I've added library for the loaders